server/schedule/checker/replica_checker.go

```diff
@@ -62,37 +64,49 @@ func (r *ReplicaChecker) GetType() string {
 }
 
 // Check verifies a region's replicas, creating an operator.Operator if need.
-func (r *ReplicaChecker) Check(region *core.RegionInfo) *operator.Operator {
+func (r *ReplicaChecker) Check(region *core.RegionInfo) (op *operator.Operator) {
 	checkerCounter.WithLabelValues("replica_checker", "check").Inc()
-	if op := r.checkDownPeer(region); op != nil {
-		checkerCounter.WithLabelValues("replica_checker", "new-operator").Inc()
-		op.SetPriorityLevel(core.HighPriority)
-		return op
+	abnormalCount := 0
+	if downCount, storeID := r.checkDownPeer(region); downCount > 0 {
+		op = r.fixPeer(region, storeID, downStatus)
+		abnormalCount = abnormalCount + downCount
 	}
-	if op := r.checkOfflinePeer(region); op != nil {
-		checkerCounter.WithLabelValues("replica_checker", "new-operator").Inc()
-		op.SetPriorityLevel(core.HighPriority)
```

Comment:
Why remove it?

Reply:
not remove, it will added in next condition